src/deployment.ts

```diff
@@ -161,6 +174,7 @@ export class Deployment extends workload.Workload implements IScalable {
 
     this.replicas = props.replicas;
     this.strategy = props.strategy ?? DeploymentStrategy.rollingUpdate();
+    this.revisionHistoryLimit = props.revisionHistoryLimit;
```

Comment:
This is not enough though to get this included in the manifest. It also needs to added to `_toKube()`:

https://github.com/cdk8s-team/cdk8s-plus/blob/2be034ce6d68faeec22fe96cb8fda539aa020400/src/deployment.ts#L230-L242